scalafmt-interfaces/src/main/java/org/scalafmt/interfaces/Scalafmt.java

```diff
@@ -79,7 +79,7 @@
     Scalafmt withMavenRepositories(String ... repositories);
 
     /**
-     * Clear internal caches such as classloaded Scalafmt instances.
+     * Clear internal caches such as classloader Scalafmt instances.
```

Comment:
I'm not sure about this at all.

Google has <7000 hits for `classloaded`. There are ~350 from Oracle, so they might believe it's a real thing. The internet as a whole does not appear to subscribe.

I'd be happy to drop this.

And to be clear, I was aware of what it might be trying to say and manually chose to make this suggestion, it wasn't remotely automated.

Reply:
i'd suggest dropping it or replacing with something else, as it doesn't make sense to me anymore.

Reply:
Dropped. I've placed this commit at the beginning of the series w/ a distinctive commit message.